src/gui/FileBrowser.cpp

```diff
@@ -97,6 +98,10 @@ FileBrowser::FileBrowser(const QString & directories, const QString & filter,
 
 	addContentWidget( ops );
 
+	// Whenever the FileBrowser has focus, Ctrl+F should direct focus to its filter box.
+	QShortcut *filterFocusShortcut = new QShortcut( QKeySequence( Qt::CTRL + Qt::Key_F ), this, SLOT(giveFocusToFilter()) );
```

Comment:
Haha, I'll go ahead & replace that. I actually want to make sure that opening the File Browser doesn't steal focus in a bad way before this gets merged. I think that if you have Piano Roll open & it has keyboard focus, the current behavior might be that it maintains keyboard focus when you open the file browser, and the new behavior would be that it loses focus - I'm not sure, I'll have to test.